docs/sources/developers/plugins/metadata.md

```diff
@@ -22,25 +22,26 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
+| `builtIn`            | boolean                       | No       | Indicates whether the plugin is developed and shipped as part of Grafana. Also known as a 'core plugin'.                                                                                                                                                                                                                                                                                                |
+| `category`           | string                        | No       | Plugin category used on the "Add data source" page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `profiling`, `sql`, `enterprise`, `iot`, `other`.                                                                                                                                                                                                                                       |
+| `enterpriseFeatures` | [object](#enterprisefeatures) | No       | Grafana Enerprise specific features                                                                                                                                                                                                                                                                                                                                                                     |
 | `executable`         | string                        | No       | The first part of the file name of the backend component executable. There can be multiple executables built for different operating system and architecture. Grafana will check for executables named `<executable>_<$GOOS>_<lower case $GOARCH><.exe for Windows>`, e.g. `plugin_linux_amd64`. Combination of $GOOS and $GOARCH can be found here: https://golang.org/doc/install/source#environment. |
-| `hiddenQueries`      | boolean                       | No       | For data source plugins, include hidden queries in the data request.                                                                                                                                                                                                                                                                                                                                    |
+| `hideFromList`       | boolean                       | No       | Excludes the plugin from listings in Grafana's UI. Only allowed for `builtIn` plugins.                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Not entirely true as this one only applies to `welcome` and `gettingstarted` - again another internal field that shouldn't be used 🤦